lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -183,6 +187,25 @@ def create_or_update_user_script_phases(script_phases, native_target)
             end
           end
 
+          # Script phases can have a limited number of input and output paths due to each one being exported to `env`.
+          # A large number can cause a build failure because of limitations in `env`. See issue
+          # https://github.com/CocoaPods/CocoaPods/issues/7362 and https://bugreport.apple.com/web/?problemID=36831832
```

Comment:
Not sure if its a big deal, but that bug report link is only accessible to the account that created it so it isn't super helpful for others. Is there a radar filed that can be linked to on http://www.openradar.me ?

Reply:
good point